easybuild/easyconfigs/n/numba/llvmlite-0.27.1_fix-ffi-Makefile.patch

```diff
@@ -0,0 +1,17 @@
+Make sure easybuild flags and libs are used for llvmlite
+Author: Samuel Moors, Vrije Universiteit Brussel (VUB)
+diff -ur llvmlite-0.27.1.orig/ffi/Makefile.linux llvmlite-0.27.1/ffi/Makefile.linux
+--- llvmlite-0.27.1.orig/ffi/Makefile.linux	2019-02-01 21:16:16.000000000 +0100
++++ llvmlite-0.27.1/ffi/Makefile.linux	2019-02-14 13:38:18.147608713 +0100
+@@ -5,9 +5,9 @@
+ CXX_FLTO_FLAGS ?= -flto
+ LD_FLTO_FLAGS ?= -flto -Wl,--exclude-libs=ALL
+ 
+-CXXFLAGS = $(LLVM_CXXFLAGS) $(CXX_FLTO_FLAGS)
++CXXFLAGS := $(CXXFLAGS) $(LLVM_CXXFLAGS) $(CXX_FLTO_FLAGS)
+ LDFLAGS := $(LDFLAGS) $(LLVM_LDFLAGS) $(LD_FLTO_FLAGS)
+-LIBS = $(LLVM_LIBS)
++LIBS := $(LIBS) $(LLVM_LIBS)
```

Comment:
@smoors Is this actually needed? The right flags should get picked up via `$(LLVM_*)` (since LLVM was built with EB)?

If it is needed, to patch filename should be fixed since this doesn't relate to `ffi` at all...

Reply:
OK, I guess it is actually relevant to `ffi`, since without this patch the build fails with:

```
ld: /tmp/eb-Y1Dxjn/icpc2bkoAe.o: relocation R_X86_64_32 against `.rodata.str1.4' can not be used when making a shared object; recompile with -fPIC
ld: final link failed: Nonrepresentable section on output
make: *** [libllvmlite.so] Error 1
```
```